src/interface.jl

```diff
@@ -627,6 +824,41 @@ function Base.show(io::IO, G::LieGroup)
     return print(io, "LieGroup($(G.manifold), $(G.op))")
 end
 
+_doc_vee = """
+    vee(G::LieGroup, X)
+    vee!(G::LieGroup, c, X)
+
+Compute the vee map ``(⋅)^∨`` that maps a tangent vector `X` from the [`LieAlgebra`](@ref)
+to its coordinates with respect to a certain bvasis in the Lie algebra
```

Comment:
Maybe we should explicitly mention that we mean `LieAlgebraOrthogonalBasis`.

Reply:
Sure.